### ISSUE TITLE
Address form group on all address pages is missing error class

### ISSUE THREAD

**Issue by stuartmccoll** [heritagefund/funding-frontend#240]
The wrapping `div` element with a `class` attribute of `govuk-form-group` in all address pages is missing the conditional logic that we use to append the `govuk-form-group--error` class when the model's `errors` hash has an attribute which matches this field.

**Comment by stuartmccoll** [comment]
Fixed in #364
